cmd/neofs-node/morph.go

```diff
@@ -28,3 +36,51 @@ func initMorphComponents(c *cfg) {
 	c.cfgObject.netMapStorage = wrap
 	c.cfgNetmap.wrapper = wrap
 }
+
+func listenMorphNotifications(c *cfg) {
+	subs, err := subscriber.New(c.ctx, &subscriber.Params{
+		Log:         c.log,
+		Endpoint:    c.viper.GetString(cfgMorphNotifyRPCAddress),
+		DialTimeout: c.viper.GetDuration(cfgMorphNotifyDialTimeout), // TODO: read from config
+	})
+	fatalOnErr(err)
+
+	lis, err := event.NewListener(event.ListenerParams{
+		Logger:     c.log,
+		Subscriber: subs,
+	})
+	fatalOnErr(err)
+
+	c.workers = append(c.workers, newWorkerFromFunc(lis.Listen))
+
+	setNetmapNotificationParser(c, newEpochNotification, netmapEvent.ParseNewEpoch)
+	listenContractNotifications(c.cfgNetmap.scriptHash, lis, c.cfgNetmap.parsers, c.cfgNetmap.subscribers)
```

Comment:
Listener starts listening for notification on `Listen()` or `ListenWithError()` functions. Are they missing or it will be added on different PR when replicator will be ready? Seems like `listenContractNotifications` only set events and event handlers.

Reply:
`Listener` starts processing as application worker. I renamed function to `registerNotificationHandlers` for better clarity of its purpose.